Source/WebCore/dom/Document.cpp

```diff
@@ -2407,7 +2407,7 @@ String Document::nodeName() const
 WakeLockManager& Document::wakeLockManager()
 {
     if (!m_wakeLockManager)
-        m_wakeLockManager = makeUnique<WakeLockManager>(*this);
+        m_wakeLockManager = adoptRef(new WakeLockManager(*this));
```

Comment:
No, `WakeLockManager` should have a public `create()` function and its constructor should be private now that it is ref-counted.

Reply:
Fixed.